nginx/default

```diff
@@ -19,7 +16,8 @@ server {
   }
 
   # Redirect legacy v1 patch URLs.
-  # FIXME: This breaks download of sqlite that has not been imported to v2 because it is replaced by sqlite3
+  # FIXME: This breaks download of sqlite that has not been imported to v2 because it is replaced by sqlite3 (xclaesse)
+  # NOTE: no longer needed, we now serve v1 patch URLs directly from a static files mirror (in /var/www/v1-static, see below) (Tim, Nov 2024)
```

Comment:
If it's no longer needed, just remove it :)

Reply:
OTOH, why not just use that redirect? Files are already hosted on github, we don't need another static mirror, do we?

Reply:
As per my previous comment, if you want to make sure the files are all there including the old sqlite things I'm all for it, I'm just not planning to do that work, and I also suspect there might be some discussions otherwise surely you'd have just done it back when you added the comment.

Reply:
I don't think we should break things is what I'm saying, so if you want to enable the redirect make sure it won't break anything.

Reply:
I ran this test script inside wrapdb-v1-api-mirror repo:
```python3
import glob
import requests

for i in glob.glob('**/get_zip', recursive=True):
    parts = i.split('/')
    name = parts[2]
    version = parts[3]
    revision = parts[4]
    url = f'https://github.com/mesonbuild/wrapdb/releases/download/{name}_{version}-{revision}/{name}_{version}-{revision}_patch.zip'
    response = requests.get(url)
    if response.status_code == 200:
        print(url, 'OK')
    else:
        print(url, 'not found')
```

Missing  URLs are:
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9c-4/libjpeg_9c-4_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9c-2/libjpeg_9c-2_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9c-3/libjpeg_9c-3_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9c-1/libjpeg_9c-1_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9a-4/libjpeg_9a-4_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9a-2/libjpeg_9a-2_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/libjpeg_9a-5/libjpeg_9a-5_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3310100-1/sqlite_3310100-1_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3320300-4/sqlite_3320300-4_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3320300-3/sqlite_3320300-3_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3320300-5/sqlite_3320300-5_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3250100-2/sqlite_3250100-2_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3250100-1/sqlite_3250100-1_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3230100-1/sqlite_3230100-1_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3080802-4/sqlite_3080802-4_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3080802-2/sqlite_3080802-2_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3080802-5/sqlite_3080802-5_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3140200-2/sqlite_3140200-2_patch.zip
https://github.com/mesonbuild/wrapdb/releases/download/sqlite_3140200-1/sqlite_3140200-1_patch.zip

IMHO it's fine to break queries and download of wraps, we just need to keep patch zips. Those 2 deps are trivial to update to v2, I don't think it's a big deal to break them, it's been deprecated for years.

Reply:
Re. "trivial to update" that's beside the point because people will be consuming third party source repositories/branches/tags that they can't easily update without switching to their own modified copies or overwriting the checkouts manually.